daemon/README.md

```diff
@@ -227,11 +227,13 @@ Autogenerated D-Bus autostart configuration
   - for each application that defines:
         - OrganizationName
         - ApplicationName
-        - ExecDBus
+        - ExecDBus or X-Share-Methods
```

Comment:
Unless I'm mistaken we prefer to have ExecDBus defined also with X-Share-Methods. And that "or" could make one think that having both is not an option. Maybe "A and/or B" or "A or B  or both" or stte?

Reply:
Yeah, I think that makes a lot of sense. I also changed the text below this a bit to clarify the role of ExecDBus.